pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -39,6 +39,11 @@ public class DdaSearchTest : AbstractFunctionalTest
         public void TestDdaSearch()
         {
             TestFilesZip = @"TestFunctional\DdaSearchTest.zip";
+
```

Comment:
Do we need this commented out code?

Reply:
I left it in intentionally. I was trying to add an automatic test to check that it throws the correct exception but that turned out to be more difficult than I thought and not worth the effort. But being able to cause that kind of exception manually seems useful and relevant to that test.